src/service/helpers.js

```diff
@@ -10,6 +10,16 @@ const uniqueID = () => {
     '-' + chr4() + chr4() + chr4();
 };
 
+const formatDateWithSlash = date => {
+  const newDate = new Date(date);
+  const day = newDate.getDate();
+  const month = newDate.getMonth()+1;
+  const year = newDate.getFullYear();
+
+  return `${month}/${day}/${year}`;
+}
```

Comment:
I think the dates are in `en-GB` format. You can have this instead to format the date `new Date().toLocaleDateString("en-GB")` which outputs date in format `${day}/${month}/${year}`